photon-lib/src/main/scala/com/linkedin/photon/ml/Types.scala

```diff
@@ -24,6 +24,16 @@ object Types {
   type SSV = org.apache.spark.mllib.linalg.SparseVector
   type SparkVector = org.apache.spark.mllib.linalg.Vector
 
+  // A "sample" is a training, validation or scoring vector of feature values
+  type UniqueSampleId = Long
+
+  // A "coordinate" is made up of several random effects
```

Comment:
A coordinate is a fixed effect, random effect, or matrix factorization model - a random effect coordinate is made up of several fixed effect models, one per unique random effect ID